File: server/src/main/java/com/fluffy/exam/ui/request/UpdateExamTitleWebRequest.java
package com.fluffy.exam.ui.request;

import com.fluffy.exam.application.request.UpdateExamTitleRequest;
import com.fluffy.global.web.Accessor;

public record UpdateExamTitleWebRequest(String title) {

    public UpdateExamTitleRequest toAppRequest(Long examId, Accessor accessor) {
        return new UpdateExamTitleRequest(title, examId, accessor);
    }
}

File: server/src/main/kotlin/com/fluffy/exam/ui/request/UpdateExamTitleWebRequest.kt
package com.fluffy.exam.ui.request

import com.fluffy.exam.application.request.UpdateExamTitleRequest
import com.fluffy.global.web.Accessor

data class UpdateExamTitleWebRequest(val title: String) {

    fun toAppRequest(examId: Long, accessor: Accessor): UpdateExamTitleRequest {
        return UpdateExamTitleRequest(
            examId = examId,
            title = title,
            accessor = accessor,
        )
    }
}